cypress/integration/login.spec.js

```diff
@@ -90,3 +90,39 @@ describe('login', () => {
     cy.getCookies().should('have.length', 0);
   });
 });
+
+describe('login?loggedOut=True', () => {
+  beforeEach(() => {
```

Comment:
You can drop this `beforeEach` and `describe` and just put all of these tests in the top `describe`.

Reply:
Good point.  When I started I didn't think I was going to use the same URL for each test.